content/rss.py

```diff
@@ -14,6 +14,10 @@ def rss_entry(blog):
     ans+= "<guid isPermaLink=\"true\">"+blog["permalink"]+"</guid>"
     if "Summary" in blog:
         ans+= "<description>"+blog["Summary"]+"</description>"
+    if "Content" in blog:
+        with open(blog["Content"], "r") as f:
```

Comment:
Need to pre-pend `content/` to the file but other than that it works for me.

Reply:
Marking as resolved with latest code push. Now that I think about it, that's probably poor etiquette and you were supposed to mark it as resolved, oops lmao we ball